config/install/rules.reaction.set_the_user_to_follow_the_idea_after_joining_the_group_.yml

```diff
@@ -0,0 +1,56 @@
+langcode: en
+status: true
+dependencies: {  }
+id: set_the_user_to_follow_the_idea_after_joining_the_group_
```

Comment:
Let's use better machine names.
For example: `follow_idea_after_joining_group`.